### ISSUE TITLE
Automatic version labeling only updates on autoreconf

### ISSUE THREAD

**Issue by jangler** [schismtracker/schismtracker#77]
Latest build from my repo (and therefore the 20170420 release) still says 20161223.

**Comment by jangler** [comment]
Note that an autoreconf correctly updates the date. Is that what's supposed to happen? It's been a year and I don't remember exactly how this was supposed to work.

**Comment by asbjornu** [comment]
What I noticed in #109 is that `schismtracker --version` correctly reports `20170628`, but when the binary crashes, problem report believes that the version is `20120105`. Perhaps that's stale metadata external to the `schismtracker` binary somewhere inside the `.app` bundle?

**Comment by jangler** [comment]
Yes, that problem report is based on something in the .app bundle, unrelated to the automatic version labeling compiled into the binary.

I'm guessing this behavior is "expected" based on how the version labeling was implemented, but I think we need to move some of the logic from configure.ac or something into the final makefile (which is ultimately generated from configure.ac). It's kind of broken otherwise.

**Comment by jangler** [comment]
@mb14 do you have any input re: this issue?
